app/src/main/java/de/danoeh/antennapod/fragment/SubscriptionFragment.java

```diff
@@ -96,9 +96,9 @@ public class SubscriptionFragment extends Fragment
     private boolean displayUpArrow;
 
     private Disposable disposable;
-    private SharedPreferences prefs;
 
     private SpeedDialView speedDialView;
+    private SharedPreferences prefs;
```

Comment:
Please revert the unrelated formatting changes in this file

Reply:
Change is reverted